events_task/lib/my_app_web/templates/event/form.html.eex

```diff
@@ -0,0 +1,19 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
```

Comment:
If you wanted to show a text when there are some errors - it would be better to test for exactly it:
`if Enum.any?(@changeset.errors)`.

Changeset having an action is only a side-effect, in general it's better not to rely on side-effects when you can check something directly.